packages/platform-node-shared/test/NodeFileSystem.test.ts

```diff
@@ -64,7 +66,7 @@ describe("FileSystem", () => {
       expect(after).toEqual("")
     })))
 
-  it("should track the cursor position when reading", () =>
+  it.skipIf(isDeno)("should track the cursor position when reading", () =>
```

Comment:
What's the issue here?

Reply:
The test fails. It's a bug in Deno, we hit it in effect-platform-deno as well: https://github.com/lishaduck/effect-utils/blob/eb3bdb3b35b6ca6fe448472e37c4440df5be7bb3/packages/platform-deno/tests/DenoFileSystem.test.ts#L75-L77

I'm 99% sure it's a bug in Deno, but I didn't see any open issues the last time I checked and I didn't understand the issue well enough to file one.

Reply:
(Still need to fix tests not to run, only did tsc)

Reply:
Fixed it! (https://github.com/vitest-dev/vitest/issues/8280)